packages/composites/bitcoin-json-rpc/src/config/index.ts

```diff
@@ -1,7 +1,7 @@
 import { Requester, util } from '@chainlink/ea-bootstrap'
 import { Config } from '@chainlink/types'
 
-export const DEFAULT_ENDPOINT = 'difficulty'
+export const DEFAULT_ENDPOINT = 'getblockchaininfo'
```

Comment:
Why was this changed?

Reply:
`getblockchaininfo` is the method that is executed. 'difficulty' is the default resulth path, but if this is changed, other information can be obtained. It seemed better to me to call the endpoint as the name of the method to execute. Do you think it's OK?